drakrun/drakrun/main.py

```diff
@@ -297,7 +297,7 @@ def process(self):
             extension = 'dll'
         self.log.info("Running file as %s", extension)
 
-        file_name = self.current_task.payload.get("file_name", f"malwar.{extension}")
+        file_name = self.current_task.payload.get(f"file_name.{extension}", f"malwar.{extension}")
```

Comment:
:thinking:

Reply:
Looks like this `f"file_name.{extension}"` is not correct, key names inside `payload` dict should be always constant (i.e. static strings)

Reply:
This is ofc. totally wrong. My bad. It was supposed to be a quick fix:/